image-compare-pdfs.js

```diff
@@ -7,57 +7,86 @@ const { PNG } = require('pngjs');
-  'Notice_Of_Change_Of_Trial_Judge.pdf',
-  'Notice_Of_Change_To_In_Person_Proceeding.pdf',
-  'Notice_Of_Docket_Change.pdf',
-  'Notice_Receipt_Petition.pdf',
-  'Notice_Trial_Issued_In_Person.pdf',
-  'Notice_Trial_Issued.pdf',
-  'Notice.pdf',
-  'Order_Designating_Place_Of_Trial.pdf',
-  'Order_For_Amended_Petition.pdf',
-  'Order_For_Amended_Petition_And_Filing_Fee.pdf',
-  'Order_For_Filing_Fee.pdf',
-  'Order_To_Show_Cause.pdf',
-  'Order.pdf',
-  'Pending_Report.pdf',
-  'Practitioner_Case_List.pdf',
-  'Receipt_of_Filing.pdf',
-  'Standing_Pretrial_Order_For_Small_Case.pdf',
-  'Standing_Pretrial_Order.pdf',
-  'Trial_Calendar.pdf',
-  'Trial_Session_Planning_Report.pdf',
+  { fileName: 'Address_Label_Cover_Sheet.pdf', pageNumber: 1 },
+  { fileName: 'Case_Inventory_Report.pdf', pageNumber: 1 },
+  { fileName: 'Change_Of_Address.pdf', pageNumber: 1 },
+  { fileName: 'CourtIssuedDocumentCoverSheet.pdf', pageNumber: 1 },
+  { fileName: 'CoverSheet.pdf', pageNumber: 1 },
+  { fileName: 'StampedCoverSheet.pdf', pageNumber: 1 },
+  { fileName: 'Cover_Sheet_For_Consolidated_Cases.pdf', pageNumber: 1 },
+  { fileName: 'Docket_Record.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Of_Change_To_Remote_Proceeding.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Of_Change_Of_Trial_Judge.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Of_Change_To_In_Person_Proceeding.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Of_Docket_Change.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Receipt_Petition.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Trial_Issued_In_Person.pdf', pageNumber: 1 },
+  { fileName: 'Notice_Trial_Issued.pdf', pageNumber: 1 },
+  { fileName: 'Notice.pdf', pageNumber: 1 },
+  { fileName: 'Order_Designating_Place_Of_Trial.pdf', pageNumber: 1 },
+  { fileName: 'Order_For_Amended_Petition.pdf', pageNumber: 1 },
+  { fileName: 'Order_For_Amended_Petition_And_Filing_Fee.pdf', pageNumber: 1 },
+  { fileName: 'Order_For_Filing_Fee.pdf', pageNumber: 1 },
+  { fileName: 'Order_To_Show_Cause.pdf', pageNumber: 1 },
+  { fileName: 'Order.pdf', pageNumber: 1 },
+  { fileName: 'Pending_Report.pdf', pageNumber: 1 },
+  { fileName: 'Practitioner_Case_List.pdf', pageNumber: 1 },
+  {
+    fileName: 'Printable_Trial_Session_Working_Copy_With_Case_Notes.pdf',
+    pageNumber: 1,
+  },
+  {
+    fileName: 'Printable_Trial_Session_Working_Copy_With_Case_Notes.pdf',
+    pageNumber: 2,
+  },
+  {
+    fileName: 'Printable_Trial_Session_Working_Copy_Without_Case_Notes.pdf',
+    pageNumber: 1,
+  },
+  {
+    fileName: 'Printable_Trial_Session_Working_Copy_Without_Case_Notes.pdf',
+    pageNumber: 2,
+  },
+  { fileName: 'Receipt_of_Filing.pdf', pageNumber: 1 },
+  { fileName: 'Standing_Pretrial_Order_For_Small_Case.pdf', pageNumber: 1 },
+  { fileName: 'Standing_Pretrial_Order.pdf', pageNumber: 1 },
+  { fileName: 'Trial_Calendar.pdf', pageNumber: 1 },
+  { fileName: 'Trial_Session_Planning_Report.pdf', pageNumber: 1 },
 ];
```

Comment:
💭 instead of altering `pdfs` from an array of strings to an object where only 2 exceptions have a `pageNumber` of 2, what if you had a default `pageNumber` of 1, unless the pdf is either 'Printable_Trial_Session_Working_Copy_With_Case_Notes' or 'Printable_Trial_Session_Working_Copy_Without_Case_Notes'? that could be within the loop itself. just a thought, let me know if you wanna chat about it.